bootstrap.py

```diff
@@ -117,21 +117,14 @@
 print("01. Patched sys.path with %s" % DEVPATH)
 
 
-# Selecting the GUI toolkit: PyQt5 if installed, otherwise PySide or PyQt4
-# (Note: PyQt4 is still the officially supported GUI toolkit for Spyder)
+# Selecting the GUI toolkit: PyQt5 if installed
 if options.gui is None:
     try:
         import PyQt5  # analysis:ignore
         print("02. PyQt5 is detected, selecting")
         os.environ['QT_API'] = 'pyqt5'
     except ImportError:
-        try:
-            import PyQt4  # analysis:ignore
-            print("02. PyQt4 is detected, selecting")
-            os.environ['QT_API'] = 'pyqt'
-        except ImportError:
-            print("02. No PyQt5 or PyQt4 detected, using PySide if available "
-                  "(deprecated)")
+        print("02. No PyQt5 detected (deprecated)")
```

Comment:
What does ``(deprecated)`` refer to here? Before it referred to the deprecated ``pyside`` support but that was deleted, so since PyQt5 is (hopefully) not deprecated also we can presumably delete that parenthetical?

Reply:
Sorry, I have to remove that deprecated.